protobuf_to_dict/convertor.py

```
@@ -205,7 +205,13 @@ def _dict_to_protobuf(pb, value, type_callable_map, strict, ignore_none):
                     if value_field.cpp_type == FieldDescriptor.CPPTYPE_MESSAGE:
                         _dict_to_protobuf(getattr(pb, field.name)[key], value, type_callable_map, strict, ignore_none)
                     else:
-                        getattr(pb, field.name)[key] = value
+                        if ignore_none and value is None:
+                            continue
+                        try:
+                            getattr(pb, field.name)[key] = value
+                        except Exception as e:
+                            print(f"type: {type(pb)}, field: {field.name}, value: {value}. Error: {e}")
```

Why are we printing instead of logging? If we don't have the logger, we can pass it to the class.

---

Or we can re-raise the exception and add all these information to the message of the error.